Breadth_First_Search/BFS_in_Csharp.cs

```diff
@@ -0,0 +1,137 @@
+//Breadth First Search ALgorithm implemented using queue in C# 
+
+using System;
+// for generic data structures  
+using System.Collections.Generic;                        
+
+namespace GraphBfs
+{
+    class graph
+    {   //no of vertices in a graph
+        private int vertices;   
+        //declaraing list
+        private List<Int32>[] adj;                      
+        public graph(int v)                    
+        {   
+            //initialising Adjacency List with number of vertices
+            vertices=v;
```

Comment:
space around =

Reply:
@singh-shreya6  All changes have been done. Kindly look into it. Thanks